File: src/main/scala/analysis/advertising/bean/AdStat.scala
package analysis.advertising.bean

/**
 * 广告实时统计
 *
 * @param date
 * @param province
 * @param city
 * @param adid
 * @param clickCount
 */
case class AdStat(date: String,
                  province: String,
                  city: String,
                  adid: Long,
                  clickCount: Long)

File: src/main/java/com/jxust/entity/AdStat.java
package com.jxust.entity;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
* Create by Jayden in 2020/3/7
*/

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AdStat implements Serializable {
    private String date;

    private String province;

    private String city;

    private Integer adid;

    private Integer clickcount;

    private static final long serialVersionUID = 1L;
}